ecc-functions/owners-transfercontext/__init__.py

```
@@ -0,0 +1,22 @@
+import os, json, urllib.request, urllib.parse, azure.functions as func
+
+SUPABASE_URL = os.environ.get("SUPABASE_URL")
+SUPABASE_KEY = os.environ.get("SUPABASE_SERVICE_ROLE_KEY")
+
+def _sb(table, params):
+    qs = urllib.parse.urlencode(params, doseq=True)
+    url = f"{SUPABASE_URL}/rest/v1/{table}?{qs}"
+    req = urllib.request.Request(url, headers={"apikey": SUPABASE_KEY, "Authorization": f"Bearer {SUPABASE_KEY}"})
```

Ruff (0.13.1)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)